### ISSUE TITLE
Alignment issues 

### ISSUE THREAD

**Issue by musm** [cormullion/juliamono#78]
I'm not sure if this is even fixable in the sense that it should be fixed, but I do find this bothersome:

![image](https://user-images.githubusercontent.com/4319522/98325246-58db1e80-1fbc-11eb-8f76-169c232a07fe.png)

**Comment by cormullion** [comment]
Can't reproduce here:

<img width="823" alt="Screenshot 2020-11-06 at 08 39 10" src="https://user-images.githubusercontent.com/52289/98344743-987d2700-200b-11eb-89cc-dba70f707162.png">

To fix it, you could switch to MacOS. 😄

Joking aside, this debate has been going on for over a decade, and many pixels have suffered in endless arguments between the Microsoft Way and the Apple Way of displaying type on screen. [This article](https://damieng.com/blog/2007/06/13/font-rendering-philosophies-of-windows-and-mac-os-x) is just the first thing I found. The author later points out that "It’s a shame neither Microsoft or Apple give their users the choice."

**Comment by cormullion** [comment]
Can't reproduce here:

<img width="780" alt="Screenshot 2020-11-06 at 08 57 07" src="https://user-images.githubusercontent.com/52289/98346578-12aeab00-200e-11eb-85fe-63a792371585.png">

More info required... :)

**Comment by musm** [comment]
hmm very interesting, let me double check. Perhaps it's a terminal aliasing issue

**Comment by musm** [comment]
One issue was that I was using the semi-light variant. Anyways, fixed that but it still persisted. I double checked aliasing, and the terminal uses `grayscale` aliasing (so I think it should be fine). Then I changed font sizes to see if the issue persisted across all sizes. It's essentially the worst at the smallest font size (coincidentally the size I prefer), then it disappears for a bit, the at the largest sizes theirs a slight difference in weight of the horizontal bars, but that shouldn't be a problem. 
![2](https://user-images.githubusercontent.com/4319522/98394105-472f6080-2028-11eb-8e00-fbfde706a5c3.gif)

**Comment by cormullion** [comment]
What OS platform is this?

**Comment by musm** [comment]
This is on a Windows Machine.

**Comment by musm** [comment]
Strangely enough, I don't see the issue when I use VS Code

**Comment by musm** [comment]
It might be a rendering bug with the Windows Terminal, although what exactly I don't know.

**Comment by cormullion** [comment]
There's two issues here - the lack of vertical alignment and the slight difference in weight. I've tweaked the latter for the next release... :)  But as for the former...

It's well-known that there are two different styles of font rendering - the Apple way, and the Microsoft way. Apple and Microsoft have always disagreed in how to display fonts on screens. The argument has been going on for over a decade, now. I found this: https://damieng.com/blog/2007/06/13/font-rendering-philosophies-of-windows-and-mac-os-x - just the tip of the iceberg.

tldr:  Windows stretches the glyph shapes to better hit pixel boundaries, but at the expense of distorting the forms. Apple renders the glyph shapes precisely, but uses antialiasing to smooth the outlines.

**Comment by musm** [comment]
@cormullion Many thanks. I am quickly going over the linked blog post, but it seems not relevant in this case. Please not cleartype is disabled both in the Windows Terminal and VS Code, so I don't believe we are seeing issues related to that in this instances. (I confirmed grayscale aliasing in both cases).

**Comment by musm** [comment]
why stale bot

**Comment by cormullion** [comment]
wait for 7 days and it will try again! 😂😂😂

**Comment by cormullion** [comment]
wait for 7 days and it will try again! 😂😂😂

**Comment by musm** [comment]
still an issue, I'll post more repro's later and comparisons to other fonts
